src/inc/fixuppointer.h

```diff
@@ -64,6 +64,20 @@ class RelativePointer
         return dac_cast<PTR_TYPE>(base + m_delta);
     }
 
+    // Returns value of the embedded encoded pointer with DPTR struct base. Assumes that the pointer is not NULL.
+    template<typename TYPE_T>
+    PTR_TYPE GetValueEmbedded(DPTR(TYPE_T) pStructBase) const
+    {
+        TYPE_T *pHostStructBase = (TYPE_T *) pStructBase;
```

Comment:
`this` pointer always points into field of structure, that corresponds to DPTR `pStructBase`. Relative offset of field in structure is constant and so `TADDR offset = ((uintptr_t) this) - ((uintptr_t) pHostStructBase);` is the same constant, except if there could be several host copies for DPTR `pStructBase`. Could structure, which correponds to DPTR `pStructBase`, be copied to several places on host?